src/extension.ts

```diff
@@ -53,6 +53,12 @@ export function activate(context: vscode.ExtensionContext) {
       });
   context.subscriptions.push(disposableOneConfigurationSettings);
 
+  let disposableShowCodelens = vscode.commands.registerCommand('onevscode.show-codelens', () => {
+    let codelensState = vscode.workspace.getConfiguration('one-vscode').get('enableCodelens', true);
+    vscode.workspace.getConfiguration('one-vscode').update('enableCodelens', !codelensState, true);
```

Comment:
I think this line is for `toggle the user setting variable.`
That is, whenever user runs `show-codelens` comamnd, this will show -> hide -> show...
Am I correct?
If so, the command `show-codelens` is to show, not toggle.
It would be better to reflect the action for this command to `onevscode.toggle-codelens`
(As I don't have much experience with codelens and the terms, please let me know if this looks odd)
Another way is to provide `hide-codelens` command too, to hide only and set `show-codelens` to only show.

Reply:
and same to `"title": "show codelens"` in package.json file.

Reply:
> Am I correct?

You understood correctly. 
I think it would be more comfortable to have only one command.
So I'll change the command name.